MassTransit.BenchmarkDotNet/MassTransit.BenchmarkDotNet.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
There is already a MassTransit.Benchmark in `/tests` you should add this to

Reply:
Sorry, I must have missed that one!
I have taken the liberty of adding it to `MassTransit.sln`, as well as updating `MassTransit.BenchmarkConsole` to use `BenchmarkSwitcher`.
This will, among other things, allow the developer to run one ore more benchmarks without having to recompile the project.